.prettierignore

```diff
@@ -0,0 +1,2 @@
+*
+!(cli/src|extensions)
```

Comment:
I suppose this means ignore everything by default, but then whitelist `./cli/src` and `./extensions`?

Which additional files would get formatted without this ignore file?

Reply:
`prettier` doesn't work _only_ on js/ts files but also on a host of other formats, among which markdown. Having this whitelist means we can safely run `prettier --write .` without risking a billion lines long diff going forward as the only files we actually _want_ to be formatted are in one of those two directories. This intentionally keeps out stuff such as test fixtures and documentation.

Reply:
Have you actually tested this though? Does it actually format markdown by default?

Reply:
I have, it does. Also, as @maxrake mentioned above, we want to explicitly avoid being able to have someone set their IDE up in any way that could lead them to inadvertently format files we don't want to have formatted.